cogs/gatekeep/gatekeep.py

```diff
@@ -355,7 +316,118 @@ async def listWords(self, ctx: Context):
             pageList.append(embed)
         await menu(ctx, pageList, DEFAULT_CONTROLS)
 
-    @_gatekeep.command(name="watchlist", aliases=["wl", "users"])
+    @user.command(name="initialize", aliases=["init"])
+    @commands.guild_only()
+    @checks.mod_or_permissions(administrator=True)
+    async def initWatchList(self, ctx: Context):
+        """Initialize the list of user IDs to place on the watchlist.
+        Users that joined the server for less than X days will be placed on the watchlist. (X is configurable)
+        """
+
+        def check(msg: discord.Message):
+            return msg.author == ctx.author and msg.channel == ctx.channel
+
+        # Get confirmation before initializing. It loops through every member in a server, so it could take a while
+        await ctx.send(warning("This operation may take a while. Type 'yes' to confirm."))
+        try:
+            response = await self.bot.wait_for("message", timeout=30.0, check=check)
+        except asyncio.TimeoutError:
+            await ctx.send("No response after 30 seconds, this operation will not be executed.")
+            return
+
+        if response.content.lower() != "yes":
+            await ctx.send("This operation will not be executed.")
+            return
+
+        # Confirmed, so initialization process begins
+        start = time.time()
+        current = datetime.now(timezone.utc)
+        nDays = await self.config.guild(ctx.guild).get_attr(KEY_NEW_USER_DAYS)()
+        watchList = []
+        for member in ctx.guild.members:
+            # If a member has been in the server for less than X days, then they get added to the watch list (X is configurable)
+            if (
+                current - member.joined_at < timedelta(days=nDays)
+                and not member.guild_permissions.administrator
+                and not await self.bot.is_automod_immune(member)
+            ):
+                watchList.append(int(member.id))
+                self.logger.info(
+                    "%s#%s (%s) added to the watch list.",
+                    member.name,
+                    member.discriminator,
+                    member.id,
+                )
+        await self.config.guild(ctx.guild).get_attr(KEY_WATCH_LIST).set(watchList)
+
+        end = time.time() - start
+        await ctx.send(f"Operation took {end:.3f} seconds.")
+
+    @user.command(name="add")
+    @commands.guild_only()
+    @checks.mod_or_permissions(administrator=True)
+    async def addUser(self, ctx: Context, id: int):
```

Comment:
Can you use `user: discord.User` here instead, and then store `user.id` down below?

Reply:
If you pass a Discord ID here, it'll get converted to a discord.User object.